libcodechecker/server/api/report_server.py

```diff
@@ -13,12 +13,15 @@
 from datetime import datetime, timedelta
 import hashlib
 import json
+import os
 import sys
 import tempfile
 import zipfile
 import zlib
 
 import sqlalchemy
+from sqlalchemy.sql.expression import or_, and_, func, \
```

Comment:
Do we need this change and the `import os` above? I do not see any usages in the changes but I might miss something.

Reply:
The previous import with `*` imported the `os` too that is why we could use it.
Importing `*` is a bad practice because you do not know precisely what is imported.

It is the same case with importing `datetime`.